kernel/CMakeLists.txt

```diff
@@ -94,6 +94,16 @@ list(APPEND kernel_files
 endif()
 
 add_library(kernel ${kernel_files})
+target_link_libraries(kernel c_picolibc)
+target_link_libraries(kernel c_common)
+
+add_library(c INTERFACE IMPORTED GLOBAL)
+add_library(gcc INTERFACE IMPORTED GLOBAL)
```

Comment:
technically `GLOBAL` is superfluous here, but has been added until it has been decided where to actually place this lib import.